charts/osm/Chart.yaml

```diff
@@ -21,7 +21,7 @@ version: 0.9.1
 appVersion: v0.9.1
 
 # This specifies the minimum Kubernetes version OSM is compatible with.
-kubeVersion: ">= 1.19.0"
+kubeVersion: ">= 1.19.0-rc0"
```

Comment:
Reading through https://github.com/helm/helm/issues/3810 it appears as though the pre-release string can be anything. Using `rc0` is very specific and could leave other readers wondering if there is some logic that checks for common pre-release strings, which it appears there is not. My preference would be to use `-0` as it is a common logical base.

Reply:
Makes sense. I've updated it to `-0`